client/app.jsx

```diff
@@ -41,6 +45,12 @@ export default class App extends React.Component {
     this.setState({ user });
   }
 
+  handleSignOut() {
+    window.localStorage.removeItem('user-jwt');
+    window.localStorage.removeItem('userId');
```

Comment:
You don't need this userId in localstorage, please remove it from all of your code. Look for it every place where you are touching localstorage. If you think you need it somewhere, please show me and we can fix the code so you don't need it.